[2025-05-03T14:58:21.021+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-03T14:58:21.037+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: random_user_dag.random_user_etl_pipeline manual__2025-05-03T14:58:19.034408+00:00 [queued]>
[2025-05-03T14:58:21.045+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: random_user_dag.random_user_etl_pipeline manual__2025-05-03T14:58:19.034408+00:00 [queued]>
[2025-05-03T14:58:21.045+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 1
[2025-05-03T14:58:21.059+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): random_user_etl_pipeline> on 2025-05-03 14:58:19.034408+00:00
[2025-05-03T14:58:21.066+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2667) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-03T14:58:21.068+0000] {standard_task_runner.py:72} INFO - Started process 2669 to run task
[2025-05-03T14:58:21.069+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'random_user_dag', 'random_user_etl_pipeline', 'manual__2025-05-03T14:58:19.034408+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/air.py', '--cfg-path', '/tmp/tmpk8v8i0qv']
[2025-05-03T14:58:21.071+0000] {standard_task_runner.py:105} INFO - Job 16: Subtask random_user_etl_pipeline
[2025-05-03T14:58:21.120+0000] {task_command.py:467} INFO - Running <TaskInstance: random_user_dag.random_user_etl_pipeline manual__2025-05-03T14:58:19.034408+00:00 [running]> on host d48b01d101a5
[2025-05-03T14:58:21.213+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='random_user_dag' AIRFLOW_CTX_TASK_ID='random_user_etl_pipeline' AIRFLOW_CTX_EXECUTION_DATE='2025-05-03T14:58:19.034408+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-03T14:58:19.034408+00:00'
[2025-05-03T14:58:21.215+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-05-03T14:58:29.557+0000] {python.py:240} INFO - Done. Returned value was: done
[2025-05-03T14:58:29.584+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-05-03T14:58:29.585+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=random_user_dag, task_id=random_user_etl_pipeline, run_id=manual__2025-05-03T14:58:19.034408+00:00, execution_date=20250503T145819, start_date=20250503T145821, end_date=20250503T145829
[2025-05-03T14:58:29.619+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2025-05-03T14:58:29.634+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-03T14:58:29.637+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
